tests/mutex/LockMutexTest.php

```diff
@@ -106,4 +104,41 @@ public function testUnlockFailsAfterException()
             throw new \DomainException();
         });
     }
+
+    /**
+     * Tests the code result is available in LockReleaseException.
+     */
+    public function testCodeResultAvailableAfterFailedUnlock()
+    {
+        $this->mutex->expects($this->once())
+            ->method("unlock")
+            ->willThrowException(new LockReleaseException());
+
+        try {
+            $this->mutex->synchronized(function () {
+                return "result";
+            });
+        } catch (LockReleaseException $exception) {
+            $this->assertEquals("result", $exception->getCodeResult());
+        }
+    }
+
+    /**
+     * Tests the code exception is available in LockReleaseException.
+     */
+    public function testCodeExceptionAvailableAfterFailedUnlock()
+    {
+        $this->mutex->expects($this->any())
```

Comment:
This should be `$this->once()` instead of `$this->any()`.

Reply:
Changed. I based this method on `testUnlockFailsAfterException`, so I changed it there too.